src/apiActionSteps.ts

```diff
@@ -277,19 +274,35 @@ When(
  * @param key key of the remembered value
  */
 When(
-    'I send {string} request and save response as {string} to {string} with headers {string} with Body:',
-    async function (method: string, key: string, url: string, headers: any, body: any) {
-      const requestUrl = await memory.getValue(url);
-      const requestBody = await memory.getValue(body);
-      const requestHeaders = await memory.getValue(headers);
-      const conf: RequestInit = {
-        method,
-        body: requestBody,
-        headers: requestHeaders
-      };
-      const response = await sendHttpRequest(requestUrl, conf);
-      // store response to memory to be able to use it in next steps
-      memory.setValue(key, response);
-    },
+  'I send {string} request and save response as {string} to {string} with headers {string} with Body:',
+  async function (method: string, key: string, url: string, headers: any, body: any) {
+    const requestUrl = await memory.getValue(url);
+    const requestBody = await memory.getValue(body);
+    const requestHeaders = await memory.getValue(headers);
+    const conf: RequestInit = {
+      method,
+      body: requestBody,
+      headers: requestHeaders,
+    };
+    const response = await sendHttpRequest(requestUrl, conf);
+    // store response to memory to be able to use it in next steps
+    memory.setValue(key, response);
+  },
 );
 
+/**
+ * MANDATORY STEP THAT SHOULD BE USED AFTER SENDING REQUEST
+ * Parsing body in needed way
+ *
+ * @example
+ * I parse "$response" body as "json"
+ *
+ * @param response key of the remembered response
+ * @param type response body parsing type (arrayBuffer|formData|blob|json|text)
+ */
+When('I parse {string} body as {bodyParsingType}', async function (response: string, type: string) {
```

Comment:
I would keep type as string instead of custom as it supports quotes and double quotes

Reply:
And we need to update api-service dependency, currently it still targeting to 0.0.3

Reply:
updated version of the service.

Don't think that use `string` as a parameter type for that variable is the best way to control it. From my point of view if we use custom parameter type than we avoid possible issue of typo or wrong body parser usage that we are not supporting.

Reply:
resolved